x-pack/metricbeat/module/aws/utils.go

```diff
@@ -84,3 +84,59 @@ func GetMetricDataResults(metricDataQueries []cloudwatch.MetricDataQuery, svc cl
 func EventMapping(input map[string]interface{}, schema s.Schema) (common.MapStr, error) {
 	return schema.Apply(input, s.FailOnRequired)
 }
+
+// CheckTimestampInArray checks if input timestamp exists in timestampArray and if it exists, return the position.
+func CheckTimestampInArray(timestamp time.Time, timestampArray []time.Time) (exists bool, index int) {
+	exists = false
```

Comment:
I personally would prefer not to have named returns in this case here as for me it would make it more readable. I directly would see that the end -a, false is return and in between i, true. It would even make the code shorter.

Reply:
Yes, I need to pay attention to this next time... I got into a habit of having named returns...